src/shared/layout/MainContainer.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
 
-export const MainContainer = ({ children }) => {
-  return <div className="w-4/5 mx-auto">{children}</div>;
+export const MainContainer = ({ className, children }) => {
+  return <div className={`${className} w-4/5 mx-auto`}>{children}</div>;
```

Comment:
Can you replace the className with the literal value of `mt-20` here? Container like this shouldn't have dynamic styles. Also adjust the other pages once you put mt-20 here.